Rubberduck.VBEditor.VB6/SafeComWrappers/VB/CommandBarButtonEvents.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+using VB = Microsoft.Vbe.Interop.VB6;
+
+namespace Rubberduck.VBEditor.SafeComWrappers.VB6
+{
+    public class CommandBarButtonEvents : SafeComWrapper<VB.CommandBarEvents>, ICommandBarButtonEvents, IEventSource<VB.CommandBarEvents>
+    {
+        public CommandBarButtonEvents(VB.CommandBarEvents target, bool rewrapping = false)
+            : base(target, rewrapping)
+        {            
+        }
+
+        public VB.CommandBarEvents EventSource => Target;
```

Comment:
This would be better as an explicit interface implementation to make it less visible.